src/index.js

```diff
@@ -1,5 +1,9 @@
-import onready from './app.js';
+// import onready from "./app.js";
 import './assets/styles.css';
 import './assets/popup.css';
+import displayCards from './displayCards.js';
 
-window.addEventListener('DOMContentLoaded', onready);
+window.addEventListener('DOMContentLoaded', () => {
+  displayCards();
+  // onready();
```

Comment:
Kindly consider removing those commented lines for now and you may import them later when it's time to use them.